From: http://xenbits.xensource.com/linux-2.6.18-xen.hg?rev/6669e480cb59
# HG changeset 821 patch
# User Keir Fraser <keir.fraser@citrix.com>
# Date 1237300051 0
# Node ID 6669e480cb59e70767d835bbd0294ed5266692a9
# Parent  8b86d11a6eb39550304ab118dcade4902927a1c8
Subject: xenbus: Add new states to xenbus_strstate()

This patch adds the following states to the xenbus_strstate().
- XenbusStateReconfiguring
- XenbusStateReconfigured

Signed-off-by: Noboru Iwamatsu <n_iwamatsu@jp.fujitsu.com>
Acked-by: jbeulich@novell.com

--- sle11-2009-03-24.orig/drivers/xen/xenbus/xenbus_client.c	2008-11-25 12:35:56.000000000 +0100
+++ sle11-2009-03-24/drivers/xen/xenbus/xenbus_client.c	2009-03-17 15:27:31.000000000 +0100
@@ -53,6 +53,8 @@ const char *xenbus_strstate(enum xenbus_
 		[ XenbusStateConnected    ] = "Connected",
 		[ XenbusStateClosing      ] = "Closing",
 		[ XenbusStateClosed	  ] = "Closed",
+		[ XenbusStateReconfiguring ] = "Reconfiguring",
+		[ XenbusStateReconfigured ] = "Reconfigured",
 	};
 	return (state < ARRAY_SIZE(name)) ? name[state] : "INVALID";
 }
